server/api/cities/index.js

```diff
@@ -50,6 +50,10 @@ router.delete( '/:cityName/businesses/:businessId',  deleteCityBusiness );
 //  Route Functions
 // *****************************************************************************
 
+function hola( saludo ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "saludo". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube:9000/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)